include/sound/sof/info.h

```diff
@@ -74,11 +75,12 @@ enum sof_ipc_region {
 };
 
 struct sof_ipc_ext_data_hdr {
-	struct sof_ipc_hdr hdr;
+	struct sof_ipc_cmd_hdr hdr;
 	uint32_t type;		/**< SOF_IPC_EXT_ */
 } __packed;
 
 struct sof_ipc_dma_buffer_elem {
+	struct sof_ipc_hdr hdr;
```

Comment:
same here, you use sof_ipc_cmd_hdr to pass the SOF_IPC_EXT info, but below you use sof_ipc_hdr to pass the SOF_IPC_REGION_ info. Why is this different?

Reply:
sof_ipc_dma_buffer_elem and friends are sub structures, all the parent boot info has sof_ipc_ext_data_hdr